File: 프로그래머스/0/181899. 카운트 다운/카운트 다운.java
import java.util.ArrayList;
import java.util.List;

class Solution {
    public List<Integer> solution(int start_num, int end_num) {
        List<Integer> answer = new ArrayList<>();
        
        for(int i = start_num; i >= end_num; i--) {
            answer.add(i);
        }
        return answer;
    }
}

File: 프로그래머스/0/181899. 카운트 다운/카운트 다운.py
def solution(start_num, end_num):
    # 1. range 함수를 사용해서 start_num부터 end_num까지 감소하는 범위 생성
    # 2. list로 변환 
    return list(range(start_num, end_num - 1, -1))